src/Container/PdoSnapshotStoreFactory.php

```diff
@@ -61,11 +63,13 @@ public function __invoke(ContainerInterface $container): PdoSnapshotStore
         $config = $this->options($config, $this->configId);
 
         $connection = $container->get($config['connection_service']);
+        $serializer = $container->has($config['serialize']) ? $container->get($config['serialize']) : new CallbackSerializer(null, null);
```

Comment:
`$container->has()` check can be removed, the serializer is always provided through defaultConfig, if the container is wrongly configured, an exception is thrown, that's good enough.